examples/tests/invalid/print_invalid.w

```diff
@@ -0,0 +1,8 @@
+print("hey").get("x");
+// ^^^^^^^^^ Expression must be a class or resource instance to access property "get", instead found type "void"
+
+let x = "my name is ${print("mister cloud")}";
+//      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Expected type to be one of "str,num", but got "void" instead
+
+let x = 5 + print("hello");
+//          ^^^^^^^^^^^^^^ Expected type to be "num", but got "void" instead
```

Comment:
Maybe change the test name to something like "can't use void in expressions" and add another test that verifies the same thing with a custom function (not `print`):
```wing
returns_nothing = () {};
let x = returns_nothing();
```

Reply:
Good idea

Reply:
BTW, found a bug, as it was possible to assign void to a variable - fixed and added it to the example.